demos/checkbox.html

```diff
@@ -163,27 +190,141 @@
       </section>
 
       <section class="example mdc-theme--dark">
-        <h2>Dark Theme</h2>
-        <div class="mdc-form-field">
-          <div id="mdc-checkbox-dark" class="mdc-checkbox mdc-checkbox--theme-dark">
-            <input type="checkbox"
-                   id="checkbox-dark"
-                   class="mdc-checkbox__native-control"/>
-            <div class="mdc-checkbox__background">
-              <svg class="mdc-checkbox__checkmark"
-                   viewBox="0 0 24 24">
-                <path class="mdc-checkbox__checkmark__path"
-                      fill="none"
-                      stroke="white"
-                      d="M1.73,12.91 8.1,19.28 22.79,4.59"/>
-              </svg>
-              <div class="mdc-checkbox__mixedmark"></div>
+        <h2>Dark Theme (mdc-theme--dark) - CSS Only</h2>
+        <div>
+          <div class="mdc-form-field">
+            <div id="mdc-checkbox-dark" class="mdc-checkbox">
+              <input type="checkbox"
+                     id="checkbox-dark"
+                     class="mdc-checkbox__native-control"/>
+              <div class="mdc-checkbox__background">
+                <svg class="mdc-checkbox__checkmark"
+                     viewBox="0 0 24 24">
+                  <path class="mdc-checkbox__checkmark__path"
+                        fill="none"
+                        stroke="white"
+                        d="M1.73,12.91 8.1,19.28 22.79,4.59"/>
+                </svg>
+                <div class="mdc-checkbox__mixedmark"></div>
+              </div>
+            </div>
+            <label for="checkbox-dark" id="dark-checkbox-label">This is my checkbox</label>
+          </div>
+        </div>
+        <div>
+          <div class="mdc-form-field">
+            <div class="mdc-checkbox mdc-checkbox--disabled">
+              <input type="checkbox"
+                     id="basic-disabled-checkbox"
```

Comment:
The ID `basic-disabled-checkbox` is already being used above. Rename this (and the associated `<label>` below) to something else to avoid collisions.